src/memory_region.rs

```diff
@@ -55,21 +55,28 @@ impl MemoryRegion {
     /// Does not perform a lower bounds check, as that is already done by the binary search in MemoryMapping::map()
     pub fn vm_to_host<E: UserDefinedError>(
         &self,
+        config: &Config,
         vm_addr: u64,
         len: u64,
     ) -> Result<u64, EbpfError<E>> {
         let begin_offset = vm_addr.saturating_sub(self.vm_addr);
```

Comment:
The changes here makes sense. But I've just had to re-learn how this works, which I feel like I do at least once a month. Could we add some comments about how gaps work?

Reply:
Added a in code comment.